### ISSUE TITLE
AppArgumentValueResolver with array

### ISSUE THREAD

**Issue by komex** [silexphp/Silex#1396]
AppArgumentValueResolver works incorrect with array typehint.
```php
public function testAction(array $parameters) {}
```

```
E_WARNING: class_parents(): Class array does not exist and could not be loaded
```

**Comment by f3ath** [comment]
I believe that's been fixed in #1392

**Comment by tiemevanveen** [comment]
Or with [Ints](https://github.com/silexphp/Silex/issues/1389).. but indeed fixed 💃 !
